PROJECT

```diff
@@ -6,29 +6,29 @@ domain: kci.rocks
 layout:
 - go.kubebuilder.io/v3
 projectName: db-operator
-repo: github.com/kloeckner-i/db-operator
+repo: github.com/db-operator/db-operator
 resources:
 - api:
     crdVersion: v1
   controller: true
   domain: kci.rocks
```

Comment:
will we keep the kci.rocks? or want to disconnect from the domain too?

Reply:
But then we need to decide which one should be used :)

Reply:
@db-operator/admins Do we have a domain already? I'd like not to merge without it

Reply:
@allanger yes, we can use `kinda.rocks` as domain.